CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# ImageViewer
+
+## Upcoming
+
+* Adds `reload(atIndex:)` to force reload iamge in gallery ([PR #77](https://github.com/MailOnline/ImageViewer/pull/77), [Issue #72](https://github.com/MailOnline/ImageViewer/issues/72)).
+* Fixes image appearing after async loading ([PR #76](https://github.com/MailOnline/ImageViewer/pull/76), [Issue #59](https://github.com/MailOnline/ImageViewer/issues/59)).
+* Adds option to turn off decoration views toggle by single tap.
+* Adds option to toggle swipe to dismiss horizontally.
+
+## Version 4.0
+
+* Swift 3 support.
+
+## Version 3.1
```

Comment:
Unfortunately this version is not released to Cocoapods 😢 People should use the https://github.com/MailOnline/ImageViewer/tree/swift-2.3 branch for the Swift 2.3 supported version. At this point I'm unsure if we will be able to push 3.1 to Cocoapods